modules/orca/main.tf

```diff
@@ -79,31 +78,31 @@ module "orca_workflows" {
 }
 
 
-## orca_rds - rds module
+## orca_secretsmanager - secretsmanager module
 ## =============================================================================
-module "orca_rds" {
-  source = "../rds"
+module "orca_secretsmanager" {
+  source = "../secretsmanager"
   ## --------------------------
   ## Cumulus Variables
   ## --------------------------
   ## REQUIRED
-  aws_profile       = var.aws_profile
-  lambda_subnet_ids = var.lambda_subnet_ids
-  prefix            = var.prefix
+  prefix      = var.prefix
 
   ## OPTIONAL
-  region = var.region
```

Comment:
added back to modules/lambdas/variables.tf
```
# variables needed by bootstrap lambda. Remove in ORCA-243
variable "aws_profile" {
  type        = string
  description = "AWS profile used to deploy the terraform application."
}
variable "db_deploy_arn" {
  type        = string
  description = "AWS ARN of the db_deploy lambda used to create/modify the DB."
}
variable "region" {
  type        = string
  description = "AWS region to deploy configuration to."
}
```



also moved the bootstrap lambda terraform code under modules/lambdas/main.tf
```
# Local Variables
locals {
  tags         = merge(var.tags, { Deployment = var.prefix })
  orca_buckets = [for k, v in var.buckets : v.name if v.type == "orca"]
  # Ignore aws profile in case this was deployed with CI or in a machine without
  # remove aws_profile in ORCA-243
  used_profile = contains(["${var.aws_profile == null ? "" : var.aws_profile}"], "default") ? "" : "--profile ${var.aws_profile}"
}

resource "null_resource" "bootstrap" {
  # Determine what has to change to trigger this resource being created/re-created.
  triggers = {
    bootstrap_lambda_hash = var.db_deploy_source_code_hash
  }

  # Execute the db_deploy lambda 1 time if the resource is created/re-created.
  # need to remove var.region in ORCA-243
  provisioner "local-exec" {
    command = "aws lambda invoke --function-name ${var.db_deploy_arn} ${local.used_profile} --region ${var.region} 'db_deploy-response.out'"
  }
}
```